### PR TITLE
Use smaller image assets for snippet atom

### PR DESCRIPTION
**Pull request by tomrf1**
Instead of using the master asset, use the existing breakpoint logic.

Please see the original PR for description and screenshots of the new snippet atoms: https://github.com/guardian/frontend/pull/17309

### CODE REVIEW

File: common/app/views/fragments/atoms/snippets/snippetImage.scala.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        srcset="@ImgSrc.srcsetForBreakpoint(breakpointWidth, widths.breakpoints, maybePath = None, maybeImageMedia = Some(picture), hidpi = true)" />
+        <source media="(min-width: @breakpointWidth.breakpoint.minWidth.getOrElse("0")px)"
+        sizes="@breakpointWidth.width"
+        srcset="@ImgSrc.srcsetForBreakpoint(breakpointWidth, widths.breakpoints, maybePath = None, maybeImageMedia = Some(picture))" />
+    }
+
+        <!--[if IE 9]></video><![endif]-->
+
+    <img itemprop="contentUrl" src="@ImgSrc.getFallbackUrl(picture)" />
+</picture>
```
Comment (TBonnin) [inline common/app/views/fragments/atoms/snippets/snippetImage.scala.html]: Note for @guardian/dotcom-platform: Consolidate the different image templates. They all look bizarrely quite similar 🔨 
https://github.com/guardian/frontend/blob/d7a5d4b1647848ff35e9566cc8c6eb0b9c433e34/common/app/views/fragments/image.scala.html#L37-L37
https://github.com/guardian/frontend/blob/d7a5d4b1647848ff35e9566cc8c6eb0b9c433e34/common/app/views/fragments/items/elements/facia_cards/image.scala.html

**Review by TBonnin — APPROVED** [review]
Looks ok to me. Thanks

File: common/app/views/fragments/atoms/snippets/snippetImage.scala.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+@import layout.ContentWidths.MainMedia
```
Comment (NataliaLKB) [inline common/app/views/fragments/atoms/snippets/snippetImage.scala.html]: Why can't we use https://github.com/guardian/frontend/blob/d7a5d4b1647848ff35e9566cc8c6eb0b9c433e34/common/app/views/fragments/image.scala.html ?

**Comment by NataliaLKB** [comment]
@tomrf1 can you add a fuller description? Our PR template is here: https://github.com/guardian/frontend/blob/master/.github/PULL_REQUEST_TEMPLATE.md

At least can you in your description explain what a snippet atom is and what it looks like?

**Comment by paperboyo** [comment]
If this means the master asset will be requested from our resizer at both `dpr` values – 💎!

File: common/app/views/fragments/atoms/snippets/snippetImage.scala.html (hunk 2 above)
Comment (tomrf1) [inline common/app/views/fragments/atoms/snippets/snippetImage.scala.html]: I created a simpler alternative as it was adding the `u-responsive-ratio` div, though looking again it appears I can avoid that with `isImmersiveMainMedia = true`

**Review by NataliaLKB — APPROVED** [review]
:1st_place_medal:

**Pull request merged**